src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -22,17 +24,19 @@
         + "Parameters: "
         + PREFIX_TITLE + "TITLE "
         + PREFIX_DESCRIPTION + "DESCRIPTION "
+        + PREFIX_START + "START DATE & TIME "
+        + PREFIX_END + "END DATE & TIME "
         + PREFIX_VENUE + "VENUE "
         + "[" + PREFIX_TAG + "TAG]...\n"
         + "Example: " + COMMAND_WORD + " "
-        + PREFIX_TITLE + "John Doe "
-        + PREFIX_DESCRIPTION + "98765432 "
-        + PREFIX_VENUE + "311, Clementi Ave 2, #02-25 "
-        + PREFIX_TAG + "friends "
-        + PREFIX_TAG + "owesMoney";
+        + PREFIX_TITLE + "CS2103 Final Exam "
+        + PREFIX_DESCRIPTION + "Bring calculator "
+        + PREFIX_VENUE + "MPSH4 "
+        + PREFIX_TAG + "CS2103 "
+        + PREFIX_TAG + "Final";
```

Comment:
why this example doesn't have startTime and endTime

Reply:
Added in an example for those 2